src/test/components/TrackScreenshots.test.js

```diff
@@ -115,13 +115,13 @@ describe('timeline/TrackScreenshots', function () {
     // And we should see a small screenshot hover while preview selecting.
     expect(screenShotSize).toEqual({ width: 100, height: 50 });
 
-    // Mouseup should keep this existing selection
+    // click should keep this existing selection
     const previouslySelectedPreviewSelection = getPreviewSelection(getState());
     fireEvent(
       track,
       getMouseEvent('mouseup', { pageX: LEFT + 200, pageY: TOP })
     );
-    fireEvent(track, getMouseEvent('click', { pageX: LEFT + 210, pageY: TOP }));
+    fireEvent(track, getMouseEvent('click', { pageX: LEFT + 200, pageY: TOP }));
```

Comment:
This uses the same value as the "mouseup" above. I don't know why I chose different values when I did this originally. Now the value at the `click` handler is used to compute the final preview selection size.